sdcm/tester.py

```diff
@@ -1439,6 +1444,33 @@ def run_stress_thread_bench(self, stress_cmd, duration=None, stress_num=1, keysp
             self.alter_test_tables_encryption(scylla_encryption_options=scylla_encryption_options)
         return bench_thread
 
+    def run_stress_thread_harry(self, stress_cmd, duration=None, stress_num=1, keyspace_num=1, profile=None, prefix='',
+                                # pylint: disable=too-many-arguments,unused-argument
+                                round_robin=False, stats_aggregate_cmds=True, keyspace_name=None,
+                                use_single_loader=False,
+                                stop_test_on_failure=True):  # pylint: disable=too-many-arguments,unused-argument
+
+        timeout = self.get_duration(duration)
+        stop_test_on_failure = False if not self.params.get("stop_test_on_stress_failure") else stop_test_on_failure
+
+        if self.create_stats:
+            self.update_stress_cmd_details(stress_cmd, stresser="cassandra-harry", aggregate=stats_aggregate_cmds)
+        harry_thread = CassandraHarryThread(
+            stress_cmd, loader_set=self.loaders, timeout=timeout,
+            node_list=self.db_cluster.nodes,
+            round_robin=round_robin,
+            use_single_loader=use_single_loader,
+            stop_test_on_failure=stop_test_on_failure,
+            credentials=self.db_cluster.get_db_auth()
+        )
+        harry_thread.run()
+        scylla_encryption_options = self.params.get('scylla_encryption_options')
+        if scylla_encryption_options and 'write' in stress_cmd:
```

Comment:
Is it a case for Harry to have `write` substring in `stress_cmd` ?

Reply:
No, so it's a problem.

eg: /usr/bin/cassandra-harry -run-time 2 -run-time-unit HOURS

But we may need to add support of assigning read/write opts in cassandra-harry cmdline, the wrap script was added by myself.